gallery/installing-psget.md

```diff
@@ -60,3 +64,23 @@ Exit
    Copy-Item "C:\LocalFolder\PowerShellGet\*" "$env:ProgramFiles\WindowsPowerShell\Modules\PowerShellGet\" -Recurse -Force
    Copy-Item "C:\LocalFolder\PackageManagement\*" "$env:ProgramFiles\WindowsPowerShell\Modules\PackageManagement\" -Recurse -Force
    ```
+
+#### Computers without PowerShellGet
+
+For a computer that doesn't have any version of **PowerShellGet** installed, you can download it from the [PowerShell Gallery](https://www.powershellgallery.com/).
+You'll need a computer that has **PowerShellGet** installed.
+
+1. From a computer that has **PowerShellGet** installed, download the current version of
+   **PowerShellGet**. Two folders are downloaded: **PowerShellGet** and **PackageManagement**. Each
+   folder contains a subfolder with a version number.
+
+   For more information, see [Save-Module](/powershell/module/PowershellGet/Save-Module).
```

Comment:
Move this line to under the code snippet and add a little context as to what `Save-Module` does and why a user might want additional information about it.

Reply:
Moved this to the H3 section because both sets of instructions use `Save-Module`.